### DISCUSSION TITLE
长文本行识别，偶尔会出现单个字符被识别成2个相同字符的错误，比如，只有一个"一"，识别成2个了，怎样改善？

### DISCUSSION THREAD

**Discussion by nissansz** [PaddlePaddle/PaddleOCR#12521] [Q&A]
#### 运行环境 / Runtime Environment
- OS:
- Paddle:
- PaddleOCR:

#### 附件 / Appendix

长文本行识别，偶尔会出现单个字符被识别成2个相同字符的错误，比如，只有一个"一"，识别成2个了，怎样改善？

**Comment by UserWangZz** [comment]
这是CTC解码原理存在的问题，可以更换算法或者加大训练轮数

**Comment by nissansz** [comment]
可以选哪个算法？
如果把每个字单独检测出来，最后合并识别单字结果，会不会好？

**Comment by UserWangZz** [comment]
可以了解CRNN，你说的基本上就是CTC的思想

**Comment by nissansz** [comment]
有哪个算法更好？可以改善这个问题？

**Comment by nissansz** [comment]
SVTR 可以改善？

**Comment by bely66** [comment]
@nissansz 
is this the only issue that happens, or does it also misrecognizes characters?
